### ISSUE TITLE
Support setting options

### ISSUE THREAD

**Issue by mkhoeini** [lloeki/ex-mode#8]
Support communicating with vim-mode (and probably the editor itself) by setting and getting (:set) options like 'hls', 'ts', 'ff', etc.

It probably doesn't make sense to implement a huge portion of options. However, options are a very important aspect of vim and they are just integrated into workflow of some of us. For example, if I want to temporarily turn of line numberings I just do `:set nonu` and when highlighting search matches gets in my way I do `:set nohls` and I can continue to my work without loosing the `n` command to go to the next match.

Also recently I needed to modify what motion commands like `w` recognize as word boundaries in Clojure files. In vim that is as easy as setting 'iskeyword' option in an autocmd. But in atom I don't know if that is possible at all.

**Comment by lloeki** [comment]
Good point. `:set nolist` is one I use quite often. I understant the use of `:set nohls` since I myself do `:noh` on a case by case basis for a similar use case.

**Comment by lloeki** [comment]
If applicable, setting vim options may depend on #2.

**Comment by mingelz** [comment]
I use `vim-mode-plus` now, it can highlight all matched searching words, and t9md has supported `:noh` at his ex-mode repo: https://github.com/t9md/atom-vim-mode-plus-ex-mode.
Now, t9md suggest us to use this ex-mode package, so I require `:noh` here~
